servers/Azure.Mcp.Server/README.md

```diff
@@ -118,40 +125,36 @@ npm install -g @azure/mcp
 
 ### Docker
 
-Microsoft publishes an official Azure MCP Server Docker container on the [Microsoft Artifact Registry](https://mcr.microsoft.com/artifact/mar/azure-sdk/azure-mcp).
+Pull the Docker image: [mcr.microsoft.com/azure-sdk/azure-mcp](https://mcr.microsoft.com/artifact/mar/azure-sdk/azure-mcp).
+
+```bash
+docker pull mcr.microsoft.com/azure-sdk/azure-mcp
+```
 
 <details>
-<summary>Docker setup instructions</summary>
+<summary>Docker instructions</summary>
+
+#### Create an env file with Azure credentials
 
 1. Create a `.env` file with Azure credentials ([see EnvironmentCredential options](https://learn.microsoft.com/dotnet/api/azure.identity.environmentcredential)):
 
 ```bash
-   AZURE_TENANT_ID={YOUR_AZURE_TENANT_ID}
-   AZURE_CLIENT_ID={YOUR_AZURE_CLIENT_ID}
-   AZURE_CLIENT_SECRET={YOUR_AZURE_CLIENT_SECRET}
+AZURE_TENANT_ID={YOUR_AZURE_TENANT_ID}
+AZURE_CLIENT_ID={YOUR_AZURE_CLIENT_ID}
+AZURE_CLIENT_SECRET={YOUR_AZURE_CLIENT_SECRET}
 ```
 
-2. Configure your MCP client with the Docker command:
+#### Run with environment variables
 
-```json
-   {
-      "servers": {
-         "Azure MCP Server": {
-            "command": "docker",
-            "args": [
-               "run",
-               "-i",
-               "--rm",
-               "--env-file",
-               "/full/path/to/.env",
-               "mcr.microsoft.com/azure-sdk/azure-mcp:latest"
-            ]
-         }
-      }
-   }
+2. Run using the `.env` file from step 1. Replace `/full/path/to/your.env` with the actual `.env` file path:
+
+```bash
+docker run -i --rm --env-file /full/path/to/your.env \
```

Comment:
This suggests we want customers to run this from the command line... but if we run it from the command line, it'll exit immediately.  The MCP .json file tells the MCP client how to start the server and connect to it.

Reply:
is there something wrong with the docker entry? It works as expected right now.

Reply:
How should people use Docker from outside an IDE?